lib/freeze-sys.ts

```diff
@@ -14,7 +15,8 @@ function deepFreeze (object) {
   return Object.freeze(object)
 }
 
-export default function freezeSys (obj) {
+export default function freezeSys<T>(obj: T): T {
```

Comment:
I think you could change the type to mark the `sys` portion as `Readonly` so TS is aware of the freeze, but it would be of course more of a nice to have.

Reply:
I'd rather keep it simple now to avoid unnecessary complications of types in `contentful-management`